Fix IllegalPluginAccessException occurring upon startup.

[20:57:42 ERROR]: Could not load 'plugins\Medieval-Factions-4.7.0-SNAPSHOT.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: org.bukkit.plugin.IllegalPluginAccessException: Plugin attempted to register task while disabled
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:149) ~[spigot-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:394) ~[spigot-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:301) ~[spigot-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R2.CraftServer.loadPlugins(CraftServer.java:407) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:232) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1007) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: org.bukkit.plugin.IllegalPluginAccessException: Plugin attempted to register task while disabled
	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.validate(CraftScheduler.java:468) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.runTaskTimer(CraftScheduler.java:196) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.runTaskTimer(CraftScheduler.java:187) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.scheduleSyncRepeatingTask(CraftScheduler.java:182) ~[spigot-1.18.2-R0.1-SNAPSHOT.jar:3498-Spigot-b081915-10ba1be]
	at dansplugins.factionsystem.services.ActionBarService.<init>(ActionBarService.java:20) ~[?:?]
	at dansplugins.factionsystem.MedievalFactions.<init>(MedievalFactions.java:44) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128) ~[?:?]
	at jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:347) ~[?:?]
	at java.lang.Class.newInstance(Class.java:645) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:79) ~[spigot-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:145) ~[spigot-api-1.18.2-R0.1-SNAPSHOT.jar:?]
	... 7 more

--- a/src/main/java/dansplugins/factionsystem/MedievalFactions.java
+++ b/src/main/java/dansplugins/factionsystem/MedievalFactions.java
@@ -41,7 +41,7 @@ import java.util.Arrays;
 public class MedievalFactions extends PonderBukkitPlugin {
     private final String pluginVersion = "v" + getDescription().getVersion();
 
-    private final ActionBarService actionBarService = new ActionBarService(this);
+    private final ActionBarService actionBarService = new ActionBarService();
     private final ConfigService configService = new ConfigService(this);
     private final EphemeralData ephemeralData = new EphemeralData();
     private final Logger logger = new Logger(this);
@@ -177,6 +177,7 @@ public class MedievalFactions extends PonderBukkitPlugin {
         scheduler.schedulePowerIncrease();
         scheduler.schedulePowerDecrease();
         scheduler.scheduleAutosave();
+        actionBarService.schedule(this);
     }
 
     /**

--- a/src/main/java/dansplugins/factionsystem/services/ActionBarService.java
+++ b/src/main/java/dansplugins/factionsystem/services/ActionBarService.java
@@ -16,7 +16,7 @@ import java.util.Map;
 public class ActionBarService {
     private final Map<Player, TextComponent> playerActionBarMessages = new HashMap<>();
 
-    public ActionBarService(Plugin plugin) {
+    public void schedule(Plugin plugin) {
         Bukkit.getScheduler().scheduleSyncRepeatingTask(plugin,
                 this::sendPlayerActionBarMessages, 5, 20);
     }

